package.json

```diff
@@ -1,29 +1,49 @@
 {
   "name": "hellosign-embedded",
   "version": "1.2.11",
-  "description": "A JavaScript library required for embedding HelloSign features into your webapp. For more information, see our API documentation at https://www.hellosign.com/api/documentation.",
-  "main": "src/embedded.js",
+  "description": "A JavaScript library required for using embedded signing and templates from within your application.",
+  "homepage": "https://github.com/HelloFax/hellosign-embedded",
+  "main": "embdedded.js",
```

Comment:
Was this meant to be just `embedded.js`?  I don't think there's an `embedded.js` at the root.

Reply:
Whoops, typo, and yeah this will want to be `src/embedded.js`